README.rst

```diff
@@ -0,0 +1,166 @@
+========================================================
+ The Community Earth System Model version 2.0 (CESM2.0)
+========================================================
+
+See the CESM web site for documentation and information:
+http://www2.cesm.ucar.edu
```

Comment:
http://www2.cesm.ucar.edu is no longer valid. 
This should be http://www.cesm.ucar.edu.

Reply:
Thanks, fixed.